cmd/zt_copy_s2smigration_test.go

```diff
@@ -578,6 +578,282 @@ func (s *cmdIntegrationSuite) TestS2SCopyFromContainerToContainerNoPreserveBlobT
 	})
 }
 
+func (s *cmdIntegrationSuite) TestS2SCopyFromPageToBlockBlob(c *chk.C) {
```

Comment:
IIRC, these tests are effectively unit tests of the enumeration code (i.e. they only hit code under /cmd. They don't hit code under /ste). 
Let's discuss whether to also all end-to-end (integration style) tests in test_service_to_service_copy.py.

Reply:
I'm not particularly good at python but I'll take a stab at it.

Reply:
Tests *have* been added.